src/main/java/com/zpi/domain/rest/ams/AmsServiceImpl.java

```diff
@@ -26,18 +28,18 @@
     @Override
     public boolean registerUser(User user) {
         try {
-            return client.registerUser(new UserDTO(user.getLogin(), user.getPassword()));
+            return client.registerUser(new UserDTO(user.getEmail(), user.getPassword())).getStatusCode() == HttpStatus.CREATED;
         } catch (Exception ignored) {
-            return fallback.registerUser(new UserDTO(user.getLogin(), user.getPassword()));
+            return fallback.registerUser(new UserDTO(user.getEmail(), user.getPassword())).getStatusCode() == HttpStatus.CREATED;
         }
     }
 
     @Override
     public boolean isAuthenticated(User user) {
         try {
-            return client.isAuthenticated(new UserDTO(user.getLogin(), user.getPassword()));
```

Comment:
it is fallback you shouldn't retry if your service is unavailable, log an error instead